docs/src/main/asciidoc/_high-availability/load-balancing/mod_cluster_Subsystem.adoc

```diff
@@ -250,37 +219,35 @@ discovered via the Advertise protocol or via the proxy-list attribute.
 [[remove-proxy]]
 ==== remove-proxy
 
-Remove a proxy from the discovered proxies or temporarily from the
-proxy-list attribute.
+Removes a proxy from the discovered proxies or temporarily from the static proxies list.
 
 [source,options="nowrap"]
 ----
-[standalone@localhost:9990 subsystem=modcluster] :remove-proxy(host=jfcpc, port=6666)
+[standalone@localhost:9990 /] /subsystem=modcluster/proxy=default:remove-proxy(host=localhost, port=8090)
 {"outcome" => "success"}
 ----
 
 [[add-proxy]]
 ==== add-proxy
 
-Add a proxy to the discovered proxies or temporarily to the proxy-list
-attribute.
+Adds a proxy to the discovered proxies or temporarily to the proxy-list  attribute.
```

Comment:
Should this be "or temporarily to the static proxies list"?  Matches 'remove-proxy', which sounds good.

Reply:
I rephrased this completely, hopefully that is way clearer:

> Adds/Removes a proxy from the discovered proxies at runtime without persisting to the server configuration: